builtin/game/privileges.lua

```diff
@@ -94,6 +94,10 @@ core.register_privilege("debug", {
 	give_to_singleplayer = false,
 	give_to_admin = true,
 })
+core.register_privilege("wireframe", {
+	description = "Can enable wireframe",
```

Comment:
'Can enable' is awkward English and has a double meaning, better is "Allows viewing wireframe mode".

Reply:
Oh ... just seen many priv descriptions use "Can ...", that is bad =)

Reply:
So, i am not too bothered about this being improved, up to you. We could improve all at once later.

Reply:
Comment now invalid as privs need changing.